### PR TITLE
New badge configuration

### PR DESCRIPTION
**Pull request by lpil**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

I've updated the template to use our new badges.

The README has also been formatted by the vscode markdown linter.

### Related Issues

- Part of https://github.com/EmbarkStudios/opensource-template/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,41 +116,44 @@ This will export only those Export Collections that contain _any_ of the current
...
 ### Spiral Curve
+
 Accessed from the _Embark -> Spiral_ menu item, or the 3D viewport's _Add -> Curve -> Spiral_ menu item.
 
 Adds a parametric spiral curve object to the scene.
 
 ### Connect Contextual
+
+[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4-ff69b4.svg)](../CODE_OF_CONDUCT.md)
+
```
Comment (moppius) [inline README.md:133]: This seems like the wrong place for this...

**Review by moppius — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lpil) [reply README.md:133]: It's the standard now for all Embark open source repos: https://github.com/EmbarkStudios/opensource-template/issues/20

Comment (XAMPPRocky) [reply README.md:133]: @lpil Looking at the rendered it's slightly outside of the contributing section.

Screenshot 2020-10-27 at 15 40 16

Comment (lpil) [reply README.md]: Oops! I've copy pasted this so many times now my eyes have glazed over. 😵 
It's fixed now, thank you for catching this

**Review by moppius — APPROVED** [review]

**Comment by lpil** [comment]
Thank you

**Pull request merged**